modules/map/hooks/useScreenCenter.ts

```diff
@@ -0,0 +1,24 @@
+import { useMemo } from 'react'
+import { useWindowDimensions } from 'react-native'
+import { useSafeAreaFrame } from 'react-native-safe-area-context'
+
+interface Props {
```

Comment:
Another coding style decision is we prefer `types` over `interfaces`. Please change to `type`